docs/tutorials/reality-capture.md

```diff
@@ -4,6 +4,12 @@
 const { RealityCaptureClient } = require('forge-server-utils');
 const { FORGE_CLIENT_ID, FORGE_CLIENT_SECRET } = process.env;
 const recap = new RealityCaptureClient({ client_id: FORGE_CLIENT_ID, client_secret: FORGE_CLIENT_SECRET });
-const photoscene = await recap.createPhotoScene('<scenename>', '<callback>', '<format>', '<scenetype>');
+const options = {
+    scenename: '<scene name>',
+    scenetype: SceneType.Aerial,
+    format: OutputFormat.RecapPhotoMesh,
```

Comment:
Same as above, please make sure to include the enums, e.g., `const { RealityCaptureClient } = require('forge-server-utils');`. We may also need to export these enums from the _index.ts_ file so that they're accessible.

Reply:
New commit available with fix